Containerfile

```diff
@@ -0,0 +1,29 @@
+FROM quay.io/operate-first/opf-toolbox:v0.4.1 AS bins
+ADD . /scripts
+WORKDIR /scripts
+RUN file="$(ls -1 /usr/local/bin)" && echo $file
+RUN mv  /usr/local/bin /scripts/
```

Comment:
This can be simplified into a `COPY --from` command as seen here:
https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds

Reply:
Resolved in latest commit